org.kde.kdevelop.json

```diff
@@ -0,0 +1,197 @@
+{
+    "id": "org.kde.kdevelop",
+    "runtime": "org.kde.Sdk",
+    "runtime-version": "5.15",
+    "base": "io.qt.qtwebengine.BaseApp",
+    "base-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "kdevelop",
+    "finish-args": [
+        "--allow=devel",
+        "--device=dri",
+        "--filesystem=host",
+        "--share=ipc",
+        "--share=network",
+        "--socket=cups",
+        "--socket=wayland",
+        "--socket=x11"
+    ],
+    "modules": [
+        {
+            "name": "boost",
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir -p ${FLATPAK_DEST}/include/boost",
+                "cp -r boost/* ${FLATPAK_DEST}/include/boost"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://dl.bintray.com/boostorg/release/1.74.0/source/boost_1_74_0.tar.bz2",
+                    "sha256": "83bfc1507731a0906e387fc28b7ef5417d591429e51e788417fe9ff025e116b1"
+                }
+            ]
+        },
+        {
+            "name": "grantlee",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DBUILD_TESTS=OFF"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://downloads.grantlee.org/grantlee-5.2.0.tar.gz",
+                    "sha256": "d6cd04de39a073a787c9ab8e72a0888e40779d21b83fbc788a1b36d31ac659d5"
+                }
+            ]
+        },
+        {
+            "name": "grantlee",
```

Comment:
is grantlee being in the list twice a bug? or is it on purpose? Seems like a bug but the second has a different install prefix so maybe it's on purpose?

Reply:
Fixed. Thanks!